webapp/src/components/molecules/AdvancedCard/index.tsx

```diff
@@ -1 +0,0 @@
-export { default } from './AdvancedCard';
```

Comment:
protip use `git mv $oldname $newname` so git understands it was a rename! Not required or anything just makes it a tiny bit easier to review

Reply:
Oh! Never thought of that. I'll keep that in mind next time 🙂